pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -288,6 +289,12 @@ let
       qtwayland = callPackage ../modules/qtwayland.nix {};
       qtwebchannel = callPackage ../modules/qtwebchannel.nix {};
       qtwebengine = callPackage ../modules/qtwebengine.nix {
+        # The version of Chromium used by Qt WebEngine 5.15.x does not build with clang 16 due to
+        # -Wenum-constexpr-conversion errors in v8 and incompatible function pointer conversion
```

Comment:
What about [this patch](https://gitweb.gentoo.org/repo/gentoo.git/plain/dev-qt/qtwebengine/files/qtwebengine-5.15.10_p20230623-clang16.patch?id=cabf41c23f991bab20e24a1206431f3c34ab028e)?

Reply:
Ah, forget it, I saw your comment.